.github/workflows/ios-unsigned.yml

```
@@ -0,0 +1,83 @@
+          # Build archive with code signing disabled
+          set -o pipefail
+          xcodebuild -workspace $XCODE_WORKSPACE \
+            -scheme $XCODE_SCHEME -configuration Release \
+            -destination generic/platform=iOS \
+            CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO \
+            -archivePath $ARCHIVE_PATH \
+            -resultBundlePath $RESULT_PATH \
+            clean archive | tee build-log.txt
+
+      - name: Create Payload and IPA
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:6:11: Double quote to prevent globbing and word splitting

(shellcheck)